src/components/GenreFilter.jsx

```diff
@@ -1,23 +1,16 @@
-export default function GenreFilter ( {genres, onFilterChange}) {
+export default function GenreFilter ( {genres, getGenreId}) {
```

Comment:
I think the prop name getGenreId is kinda confusing, since it isn't getting anything, it's setting it